experiments/standalone/Bucket/global_bucket_staticmap.jl

```diff
@@ -88,18 +88,9 @@ tf = 7 * 86400;
 Δt = 3600.0;
 
 # Construct albedo parameter object using static map
-# Use separate regridding directory for CPU and GPU runs to avoid race condition
-device_suffix =
```

Comment:
device_suffix is needed for doing the gpu/cpu comparison after each sim runs.

Reply:
The new regridder system makes its own folders, so we don't need this anymore

Reply:
maybe I am misunderstanding, but `device_suffix` is used in more places in these files. it is used to save the output to a file to then compare the output of the simulation (run on gpu) with the output of the simulation (run on cpu) to check that they agree

Reply:
(this has nothing to do with regridding)

Reply:
If you look at the lines below, it is only `regrid_dir` that uses `device_suffix`.

Reply:
Okay, it depends on the file. I restored it where needed.